lib/api/list-scenarios.js

```diff
@@ -35,4 +43,43 @@ module.exports = (req, res) => {
     name: config.get('name'),
     features
   });
+
+  function watchFileChanges() {
+    const socket = require('socket.io-client')(WS_URL);
+
+    // does not handle deleted files... yet...
+    chokidar.watch(path.join(codecept_dir, config.get('tests')), {ignored: /(^|[\/\\])\../}).on('all', (event, path) => {
+      console.log('Changed ', path);
```

Comment:
I like using debug() because I can turn it on when needed

Reply:
Well, this can be important user information, so I think it should be printed.